### ISSUE TITLE
Support new Count style.

### ISSUE THREAD

**Issue by jkeyes** [jkeyes/python-intercom#131]
Mirror the `for_type` approach of the Ruby library: 

```ruby
intercom.counts.for_type(type: 'conversation', count: 'admin')
intercom.counts.for_type(type: 'user', count: 'tag')
intercom.counts.for_type(type: 'user', count: 'segment')
intercom.counts.for_type(type: 'company', count: 'tag')
intercom.counts.for_type(type: 'company', count: 'user')
```

**Comment by jkeyes** [comment]
I already have this implemented, but I do need to update the integration tests.

**Comment by jkeyes** [comment]
Done and released in 3.0b3 now on [PyPi](https://pypi.python.org/pypi/python-intercom).
